collectors/ebpf.plugin/README.md

```diff
@@ -39,18 +39,93 @@ section for details.
 
 ## Charts
 
-The eBPF collector creates an **eBPF** menu in the Agent's dashboard along with three sub-menus: **File**, **VFS**, and
-**Process**. All the charts in this section update every second. The collector stores the actual value inside of its
-process, but charts only show the difference between the values collected in the previous and current seconds.
+The eBPF collector creates an **eBPF** menu in the Agent's dashboard along with two sub-menus: **Socket**, and
+**Process**. The plugin also creates charts on different menus, like System Overview, Memory, Disks, Filesystem, 
+Mount Points and Applications. All the charts in this section update every second.
+The collector stores the actual value inside of its process, but charts only show the difference between the values
+collected in the previous and current seconds.
 
-### File
+### System Overview
 
-This group has two charts demonstrating how software interacts with the Linux kernel to open and close file descriptors.
+All charts on system overview are not enabled by default, because they would consume a lot of resources.
```

Comment:
I think this sentence is not clear and could be understood (at least by me) in 2 ways:
- All charts are disabled by default
- Not all of the charts are enabled by default (some are)

Reply:
"...consume a lot of resources" sounds scary, i would never enable all the charts 😄

Reply:
ping @hugovalente-pm !

Reply:
@ilyam8 when I address the latency areas, one of the features I will bring it will be to extend the time we are reading data from kernel ring. These eBPF programs are on top of heavy operations, this is the motive we brought this warning.

Reply:
> "...consume a lot of resources" sounds scary, i would never enable all the charts

All your CPUz belong to us!

Reply:
@thiagoftsm @ilyam8 from what I understood these configurations should only be enabled if someone is trying to troubleshoot and needs to have full visibility into the date we collect with eBPF, right?

I believe the callout is needed, what we could add is an highlight to show in which case a user could want to enabled this and what would be the benefits of doing it

Reply:
I agree that warning is needed. My point is that being more specific is better - what resources (cpu, mem, etc), how many resources.

Reply:
I added some additional description trying to let it more soft, when the latest version of `libbpf` is merged during this week, we will be able to reduce the overhead, because new features will be available for us, but these features depend of kernel compilation, so we need to keep some warnings like this.

Reply:
@thiagoftsm, as @ilyam8  suggedted, can we name and, if possible, measure what are the impacts in terms of resources and a value of % of increase in usage?

Reply:
@hugovalente-pm it is possible, and I will measure to bring in the next PR, for this PR I would prefer to add this missing documentation for we have the links, in the next PR, that I expect to bring next days, I will add a table with these values.

I accepted @underhood suggestion.